package.json

```diff
@@ -80,5 +86,60 @@
       "prettier --write",
       "eslint"
     ]
+  },
+  "standard-version": {
+    "skip": {
+      "tag": true
+    },
+    "types": [
+      {
+        "type": "build",
+        "hidden": true
+      },
+      {
+        "type": "chore",
+        "hidden": true
+      },
+      {
+        "type": "ci",
+        "hidden": true
+      },
+      {
+        "type": "dev",
+        "hidden": true
+      },
+      {
+        "type": "docs",
+        "hidden": true
+      },
+      {
+        "type": "feat",
+        "section": "Features"
+      },
+      {
+        "type": "fix",
+        "section": "Bug Fixes"
+      },
+      {
+        "type": "perf",
```

Comment:
Why would we want to hide this? This feels like something worth highlighting.

Reply:
🤷‍♀️  I don't know, I'm fine with including it!